In [1]:
import numpy as np
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt

df = pd.read_csv('ForeignGifts_edu.csv',low_memory=False)
df.head()

,ID,OPEID,Institution Name,City,State,Foreign Gift Received Date,Foreign Gift Amount,Gift Type,Country of Giftor,Giftor Name
0,1,102000,Jacksonville State University,Jacksonville,AL,43738,250000,Monetary Gift,CHINA,NaN
1,2,104700,Troy University,Troy,AL,43592,463657,Contract,CHINA,Confucius Institute Headquarters
2,3,105100,University of Alabama,Tuscaloosa,AL,43466,3649107,Contract,ENGLAND,Springer Nature Customer Service Ce
3,4,105100,University of Alabama,Tuscaloosa,AL,43472,1000,Contract,SAUDI ARABIA,Saudi Arabia Education Mission
4,5,105100,University of Alabama,Tuscaloosa,AL,43479,49476,Contract,SAUDI ARABIA,Saudi Arabia Education Mission


### Brainstorm
- already know top countries donating: Qatar, Bermuda, United Arab Emirates, Canda, India, Hong Kong, England, China, Australia, France, Singapore, Saudi Arabia, Russia
- VA schools being donated by top countries: VCU (Qatar), Georgetown (Qatar)
- I would imagine UVA is comparable to Texas A&M (Qatar), U Penn(China), U of C(England)

### exploring
- who is already donating to VA
- how much
- average/median amount

In [4]:
uva_df = df[df["Institution Name"] == "University of Virginia"]
uva_df

,ID,OPEID,Institution Name,City,State,Foreign Gift Received Date,Foreign Gift Amount,Gift Type,Country of Giftor,Giftor Name
26235,26236,374500,University of Virginia,Charlottesville,VA,41822,3565,Monetary Gift,ENGLAND,West Yorkshire Police
26236,26237,374500,University of Virginia,Charlottesville,VA,41870,12265,Contract,ISRAEL,CureTech Ltd
26237,26238,374500,University of Virginia,Charlottesville,VA,41873,8187,Monetary Gift,ISRAEL,Branco Weiss Institute
26238,26239,374500,University of Virginia,Charlottesville,VA,41887,6437,Contract,ISRAEL,NovoCure Ltd
26239,26240,374500,University of Virginia,Charlottesville,VA,41904,216943,Contract,ISRAEL,InSightec-Image Guide Treatment Ltd
...,...,...,...,...,...,...,...,...,...,...
26328,26329,374500,University of Virginia,Charlottesville,VA,43602,99885,Contract,SWEDEN,AutoLiv Development AB
26329,26330,374500,University of Virginia,Charlottesville,VA,43697,1000000,Monetary Gift,SWITZERLAND,Lawrence D & Karin S Howell
26330,26331,374500,University of Virginia,Charlottesville,VA,43796,500000,Monetary Gift,GUERNSEY,Harvest Group Holding (PTC Limited)
26331,26332,374500,University of Virginia,Charlottesville,VA,43803,146810,Contract,THE NETHERLANDS,Pantarhei Oncology BV


In [6]:
uva_df.groupby("Country of Giftor")["Foreign Gift Amount"].sum().sort_values(ascending=False)

Country of Giftor
BANGLADESH         4228589
SWEDEN             4007552
TANZANIA           2776447
SWITZERLAND        2000000
ENGLAND            1950666
GUERNSEY           1500000
FINLAND            1008384
BERMUDA             900000
KOREA               862787
SINGAPORE           805000
THE NETHERLANDS     725670
ISRAEL              623655
RWANDA              284135
JAPAN               265970
FRANCE              250383
Name: Foreign Gift Amount, dtype: int64

Bangladesh, Sweden, Tanzania, Switzerland, and England are out top 5 giftors by dollar amount

In [7]:
uva_df.groupby("Country of Giftor")["Foreign Gift Amount"].size().sort_values(ascending=False)

Country of Giftor
ENGLAND            31
ISRAEL             12
BANGLADESH         10
SWEDEN             10
FINLAND             7
KOREA               6
FRANCE              5
THE NETHERLANDS     4
GUERNSEY            3
TANZANIA            3
SINGAPORE           2
SWITZERLAND         2
BERMUDA             1
RWANDA              1
JAPAN               1
Name: Foreign Gift Amount, dtype: int64

England, Israel, Bangladesh, Sweden, and Finland tope 5 giftors by count

In [8]:
giftors = uva_df.groupby("Country of Giftor")["Foreign Gift Amount"].sum().sort_values(ascending=False)
avg = giftors.mean()
med = giftors.median()

print("avg: ", avg)
print("med: ", med)

avg:  1479282.5333333334
med:  900000.0


The average for all the universities is 52202878.974842764 vs UVA's 1479282.5333333334. The median for all the universities is 6486791.5 vs UVA's 900000.0. UVA's giftors amount is low compared to the collectives.

Comparable schools: UNC, 

Institution Name
University of California, Los Angeles          3916
California Institute of Technology             2502
Columbia University in the City of New York    1127
Ohio State University (The)                    1014
Johns Hopkins University                        899
                                               ... 
University of Wisconsin - Platteville             1
Des Moines Area Community College                 1
Beth Medrash Govoha of America                    1
Columbia Southern University                      1
Presidio Graduate School                          1
Name: count, Length: 318, dtype: int64